<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="StartButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IM혜민 Bold</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>학습 시작하기</string>
   </property>
  </widget>
  <widget class="QToolButton" name="GoalButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IM혜민 Bold</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>도전 과제</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>741</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Studium">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>611</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>IM혜민 Bold</family>
      <pointsize>100</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;STUDIUM&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="OptionButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>540</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IM혜민 Bold</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>설정</string>
   </property>
  </widget>
  <widget class="QToolButton" name="InfoButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IM혜민 Bold</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>내 정보</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
